context/app/static/js/components/detailPage/visualization/Visualization/Visualization.jsx

```diff
@@ -104,67 +108,58 @@ function Visualization(props) {
       window.removeEventListener('keydown', onKeydown);
     };
   }, [collapseViz]);
+
   const isMultiDataset = Array.isArray(vitessceConfig);
 
   return (
     vitessceConfig &&
     // Don't render multi-datasets unless they have a selection from the list of options in vitessceConfig.
     (!isMultiDataset || Number.isInteger(vitessceSelection)) && (
-      <StyledPaddedSectionContainer id="visualization">
-        <StyledHeader>
-          <StyledHeaderText>Visualization</StyledHeaderText>
-          <StyledHeaderRight>
-            {hasNotebook && <VisualizationNotebookButton uuid={uuid} />}
-            <VisualizationShareButton />
-            <VisualizationThemeSwitch />
-            <SecondaryBackgroundTooltip title="Switch to Fullscreen">
-              <ExpandButton size="small" onClick={expandViz} variant="contained">
-                <FullscreenRoundedIcon color="primary" />
-              </ExpandButton>
-            </SecondaryBackgroundTooltip>
-            {isMultiDataset && (
-              <DropdownListbox
-                buttonComponent={SelectionButton}
-                optionComponent={DropdownListboxOption}
-                selectedOptionIndex={vitessceSelection}
-                options={vitessceConfig}
-                selectOnClick={setSelectionAndClearErrors}
-                getOptionLabel={(v) => v.name}
-                buttonProps={{ color: 'primary' }}
-                id="visualization-data"
-              />
-            )}
-          </StyledHeaderRight>
-        </StyledHeader>
+      <DetailPageSection id="visualization">
+        <SpacedSectionButtonRow
+          leftText={shouldDisplayHeader ? <StyledSectionHeader>Visualization</StyledSectionHeader> : undefined}
```

Comment:
```suggestion
          leftText={shouldDisplayHeader && <StyledSectionHeader>Visualization</StyledSectionHeader>}
```
If it wouldn't work, never mind.